### PR TITLE
Implementation of --repofrompath option for dnf5

### PR DESCRIPTION
**Pull request by m-blaha**
The option behave the same way as in dnf4.

Resolves: https://github.com/rpm-software-management/dnf5/issues/549
Resolves: https://github.com/rpm-software-management/dnf5/issues/168

### CODE REVIEW

**Review by jan-kolarik — APPROVED** [review]
LGTM, tested. 

Maybe we could also think of some additional CI tests. There are some existing test cases using `--repofrompath`, but it seems no test is directly testing the functionality of this option.

**Comment by m-blaha** [comment]
> LGTM, tested.
> 
> Maybe we could also think of some additional CI tests. There are some existing test cases using `--repofrompath`, but it seems no test is directly testing the functionality of this option.

Sure, I'm planning to do this. I had hoped that I will just enable current --repofrompath tests, but I found out there are none...

File: doc/dnf5.8.rst [hunk 1]
```diff
@@ -210,6 +210,15 @@ Following options are applicable in the general context for any ``dnf5`` command
     | This is a list option which can be specified multiple times.
     | Accepted values are ids, or a glob of ids.
 
+``--repofrompath=REPO_ID,REPO_PATH``
+    Specify a repository to add to the repositories only for this run. Can be used multiple times.
+
+    The new repository id is specified by ``REPO_ID`` and its baseurl by ``REPO_PATH``. Variables in both values are before creating the repo substituted.
```
Comment (kontura) [inline doc/dnf5.8.rst:216]:
```suggestion
    The new repository id is specified by ``REPO_ID`` and its baseurl by ``REPO_PATH``. Variables in both values are substituted before creating the repo.
```

**Comment by LecrisUT** [comment]
Great work on this. Can we get a minor bump? I've looked at #635, and I think only this issue is blocking the Fedora CI

**Pull request merged**